extensions/src/ACRE2Core/AmplitudeAttenuation.cpp

```diff
@@ -9,43 +9,41 @@
 
 #define AVERAGE_RATE    20
 
-int iter = 0;
 
 ACRE_RESULT CAmplitudeAttenuation::processAmplitudes(void) {
     return ACRE_OK;
 }
 
-ACRE_RESULT CAmplitudeAttenuation::process(short* samples, int sampleCount, int channels, CPlayer * player) {
-    float averageSum, currentSample, averageAmplitude, ampCoef;
-    averageSum = 0;
-    for (int i = 0; i < sampleCount*channels && i < 4095; i++, iter++) {
-        if (samples[i] > 0) {
-            currentSample = 20.0f * log10((float)samples[i]);
+ACRE_RESULT CAmplitudeAttenuation::process(int16_t *const a_samples, const int32_t ac_sampleCount, const int32_t ac_channels, CPlayer *const a_player) {
```

Comment:
Can we please not do pseudo hungarian notation? If you want to signify a variable is an argument I've always prefered a trailing _, so `player_`. Noting the constiness of an argument is also not super helpful in most modern editors/IDEs. It should show a redline.